mtags/src/main/scala-3/scala/meta/internal/pc/printer/DotcPrinter.scala

```diff
@@ -39,13 +39,33 @@ object DotcPrinter:
     def tpe(t: Type): String =
       toText(t).mkString(defaultWidth, false)
 
+    /* Overriden method because the RefinedPrinter will omit
+     * `lang` even if it's needed in case of conflicts
+     */
```

Comment:
Can you add a link to https://github.com/lampepfl/dotty/blob/75d8eea1e943bbd2c605d8411c2d52d69974d6f3/compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala#L107-L123 so we can

- tell which revision of `toTextPrefix` we borrowed
- what is the difference from the original implementation

Reply:
Done!